src/components/spinner.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
```

Comment:
Home page:
- As mentioned by the previous reviewer kindly make sure When the page loads, the web app shows the list of categories that could be filtered out by some parameter, for example by the category name.